_gitbook/conventions/README.md

```diff
@@ -0,0 +1,3 @@
+# Conventions
+
+If you follow these conventions, your code will be more accessible to other developers. If you use a standard [coding style](coding_style.md), your project will be navigable and readable to others. If you write [documentation](documenting_code.md), others will understand the intent and behavior of your code.
```

Comment:
Starting with "if you" and repeating it 3 times may not be the best approach. Maybe: "Following these conventions ...", "We encourage you to ..." 

They're more pushy, yes, but the proposed conventions here aren't very controversial. There is nothing about 80 columns, parentheses around function calls and defs, semicolons for oneline type definitions, ...